Update bots-test.md (#12142)
MicrosoftDocs/msteams-docs: msteams-platform/resources/bot-v3/bots-test.md

## Testing your bot without uploading to Teams

> Occasionally it's necessary to test your bot without installing it as an app in Teams. Testing your bot without installing it as an app can be useful to ensure your bot is available and responding, however it won't allow you to test the full breadth of Teams functionality you may have added to your bot. If you need to test your bot completely, follow the instructions for [testing by uploading](#test-by-uploading-to-teams).

Reviewer: ```suggestion
Intermittently, it's necessary to test your bot without installing it as an app in Teams to verify the bot's availability and response. However, it won't allow you to test the full breadth of Teams functionality that you might have added to your bot. To test your bot completely, follow the instructions for [testing by uploading](#test-by-uploading-to-teams).
```
